docs/containers-transports.5.md

```diff
@@ -0,0 +1,102 @@
+% CONTAINERS-TRANSPORTS(5) Containers Transports Man Page
+% Valentin Rothberg
+% April 2019
+
+
+# NAME
+containers-transports - description of supported transports for copying and storing container images
+
+
+# DESCRIPTION
```

Comment:
(Non-blocking: Having at least one sentence to say what this page is all about, and where this syntax can/can’t be used, might be useful.

Maybe
> Tools which use the containers/image library, including skopeo(1), buildah(1), podman(1), all share a common syntax for referring to container images in various locations. The general form of the syntax is _transport_`:`:_details_, where _details_ are dependent on the specified _transport_; the individual transports are documented below.

)